src/components/panes/Results/Results.tsx

```diff
@@ -8,28 +8,68 @@ import { SmallError } from '../../../icons/SmallError';
 import SomeFailed from './SomeFailed.png';
 import TestsPassed from './TestsPassed.png';
 
-const useStyles = makeStyles({
-  resultContainer: {
-    marginTop: '2em',
-    '&:not(:last-child)': {
-      paddingBottom: '2em',
-      borderBottom: '0.1em solid #8891AA',
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    header: {
+      float: 'left',
+      [theme.breakpoints.down('md')]: {
+        float: 'initial',
+      },
```

Comment:
Here I wonder if some `padding-bottom` would help to give a bit more room between the buttons and the results. It's subtle.

```
      [theme.breakpoints.down('md')]: {
        float: 'initial',
        paddingBottom: '1em'
      },
``` 

Before:
<img width="280" alt="image" src="https://user-images.githubusercontent.com/12685223/157374660-596c2965-bebc-4709-a872-44db9a0f77b3.png">

After:
<img width="282" alt="image" src="https://user-images.githubusercontent.com/12685223/157374682-8550f1aa-f1fb-4892-8627-6c28255cbfa7.png">